From 12bd229cd262f9ddd7f91bf7427e1fa5e0668da2 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 12 Dec 2019 14:15:39 +0000
Subject: [PATCH 0985/1077] MANTIS 56794 change: MDCI - Support Media Type
 "ID_DOCUMENT" codes.idl V1.5.1

added data status DS_TYPES_ID_DOCUMENT
converted tabs to spaces
fixed order of DS_TYPES




git-svn-id: svn://localhost/SelfServiceCommon/trunk@1513 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index bfd4fc08..1621ca4c 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -201,6 +201,8 @@ module datastatus
 
     const long DS_TYPES_USER_ID                  = 19400; /**< user identifaction */
 
+    const long DS_TYPES_ID_DOCUMENT              = 19500; /**< id document */
+
     const long DS_TYPES_EPASSPORT_DG1            = 20100; /**< e-Passport format */
     const long DS_TYPES_EPASSPORT_DG2            = 20200; /**< e-Passport format */
     const long DS_TYPES_EPASSPORT_DG3            = 20300; /**< e-Passport format */
@@ -241,7 +243,6 @@ module datastatus
     const long DS_TYPES_CYPHER_KEYS              = 40040; /**< encryption keys */
     const long DS_TYPES_CYPHER_TRACK             = 40050; /**< encrypted track data */
 	
-	const long DS_TYPES_ID_DOCUMENT              = 19500; /**< id document */
 };
 
 /**
-- 
2.41.0.windows.1

